packages/engines/snakemake/snakemake.aws.sh

```diff
@@ -98,38 +97,39 @@ trap "cleanup" EXIT
 
 
 # AWS Batch places multiple jobs on an instance
-# To avoid file path clobbering use the JobID and JobAttempt
+# To avoid file path clobbering use the JobID
 # to create a unique path. This is important if /opt/work
 # is mapped to a filesystem external to the container
-GUID="$AWS_BATCH_JOB_ID/$AWS_BATCH_JOB_ATTEMPT"
+GUID="$AWS_BATCH_JOB_ID"
 
 if [ "$GUID" = "/" ]; then
     GUID=`date | md5sum | cut -d " " -f 1`
 fi
 
 # Make the directory we will work in
-mkdir -p /opt/work/$GUID
-cd /opt/work/$GUID
+mkdir -p /mnt/efs/snakemake/$GUID
+cd /mnt/efs/snakemake/$GUID
 
 if [[ "$ENGINE_PROJECT" =~ ^s3://.* ]]; then
     echo "== Staging S3 Project =="
-    mkdir project
-    ENGINE_PROJECT_DIRECTORY="./project"
-    aws s3 cp --no-progress $ENGINE_PROJECT "${ENGINE_PROJECT_DIRECTORY}/" --recursive
+    ENGINE_PROJECT_DIRECTORY="."
+    echo "Copying from ${ENGINE_PROJECT} to '${ENGINE_PROJECT_DIRECTORY}/'"
+    aws s3 cp $ENGINE_PROJECT "${ENGINE_PROJECT_DIRECTORY}/"
```

Comment:
Why are we doing no progress here? I believe that we removed it because customers didn't like seeing the progress

Reply:
fixed. I removed it for debugging and didn't put it back.